src/ngx-slider/lib/slider.component.ts

```diff
@@ -441,45 +475,54 @@ export class SliderComponent implements OnInit, AfterViewInit, OnChanges, OnDest
     this.updateDisabledState();
   }
 
-  @HostListener('window:resize', ['$event'])
+  public setAriaLabel(ariaLabel: string): void {
+    this.viewOptions.ariaLabel = ariaLabel;
+    this.updateAriaLabel();
+  }
+
+  @HostListener("window:resize", ["$event"])
   public onResize(event: any): void {
     this.calculateViewDimensionsAndDetectChanges();
   }
 
-  private subscribeInputModelChangeSubject(interval?: number): void {
+  private subscribeInputModelChangeSubject(): void {
     this.inputModelChangeSubscription = this.inputModelChangeSubject
-    .pipe(
-      distinctUntilChanged(ModelChange.compare),
-      // Hack to reset the status of the distinctUntilChanged() - if a "fake" event comes through with forceChange=true,
-      // we forcefully by-pass distinctUntilChanged(), but otherwise drop the event
-      filter((modelChange: InputModelChange) => !modelChange.forceChange && !modelChange.internalChange),
-      (!ValueHelper.isNullOrUndefined(interval))
-          ? throttleTime(interval, undefined, { leading: true, trailing: true})
-          : tap(() => {}) // no-op
-    )
-    .subscribe((modelChange: InputModelChange) => this.applyInputModelChange(modelChange));
-  }
-
-  private subscribeOutputModelChangeSubject(interval?: number): void {
-    this.outputModelChangeSubscription = this.outputModelChangeSubject
       .pipe(
         distinctUntilChanged(ModelChange.compare),
-        (!ValueHelper.isNullOrUndefined(interval))
-          ? throttleTime(interval, undefined, { leading: true, trailing: true})
-          : tap(() => {}) // no-op
+        // Hack to reset the status of the distinctUntilChanged() - if a "fake" event comes through with forceChange=true,
+        // we forcefully by-pass distinctUntilChanged(), but otherwise drop the event
```

Comment:
Is this comment relevant anymore?

Reply:
Yes, it is still relevant, and it explains why the `filter()` is used below. It has to do with how I implemented change detection using `ModelChange` objects.

For a concrete example of where this is invoked, see the comment in `applyViewChange()` function where a model change is emitted with `forceChange: true`.